distributed/utils_test.py

```diff
@@ -1612,9 +1612,6 @@ def save_sys_modules():
 @contextmanager
 def check_thread_leak():
     """Context manager to ensure we haven't leaked any threads"""
-    # "TCP-Executor" threads are never stopped once they are started
-    BaseTCPConnector.warmup()
```

Comment:
Why is this not failing?

Reply:
the  threads used by `await asyncio.get_running_loop().getaddrinfo(` are shutdown when the event loop is closed